ui/src/views/Dashboard.vue

```diff
@@ -222,14 +222,22 @@
                 {{ $t("dashboard.start_migration") }}
               </button>
               <!-- other apps -->
+              <template v-else-if="app.status == 'not_migrated' || app.status == 'skipped'">
               <button
-                v-else-if="app.status == 'not_migrated'"
                 @click="showStartMigrationModal(app)"
                 :disabled="isStartMigrationButtonDisabled(app)"
                 class="btn btn-default"
               >
                 {{ $t("dashboard.start_migration") }}
               </button>
+              <button
+                  @click="toggleSkip(app)"
+                  :disabled="loading.migrationUpdate || app.status == 'syncing' || app.status == 'migrating' || accountProviderMigrationStarted"
```

Comment:
The conditions `app.status == 'syncing'` and `app.status == 'migrating'` are never reached because of `<template v-else-if="app.status == 'not_migrated' || app.status == 'skipped'">` above; I think they can be both removed